### ISSUE TITLE
Editor cannot save twice

### ISSUE THREAD

**Issue by csjx** [NCEAS/metacatui#30]
When saving a new package in the editor, on the first save, the call to `MNStorage.create()` succeeds, but the second save fails with:
``` xml
<?xml version="1.0" encoding="UTF-8"?><error detailCode="1120" errorCode="409" name="IdentifierNotUnique">
    <description>The requested identifier urn:uuid:24a449ba-d01c-4576-aca0-5c75fb4229ab is already used by another object andtherefore can not be used for this object. Clients should choosea new identifier that is unique and retry the operation or use CN.reserveIdentifier() to reserve one.</description>
</error>
```

This can be reproduced by visiting `#share`, changing the `title` clicking `Save`, changing the `abstract` and then clicking `Save` a second time.

Fix `EML211.save()` and other save functions such that they update the identifier correctly.

**Comment by csjx** [comment]
This was due to changes in `EML211.save()` where we started using `fetchSystemMetadata()` to populate the sysmeta fields from the MN. It's fixed now, but we still have issues with multiple saves.  Basically, it looks like we're causing indexing errors on the MN side because our RDF documents have multiple aggregations in them, one for each `save()` update.  So, I'm keeping this ticket open and am fixing the RDF serialization issues. I'm seeing 30 to 60 second delays on subsequent saves.
